container/libcontainer/helpers.go

```diff
@@ -104,14 +104,15 @@ func getCgroupSubsystemsHelper(allCgroups []cgroups.Mount, disableCgroups map[st
 
 // Cgroup subsystems we support listing (should be the minimal set we need stats from).
 var supportedSubsystems map[string]struct{} = map[string]struct{}{
-	"cpu":     {},
-	"cpuacct": {},
-	"memory":  {},
-	"pids":    {},
-	"cpuset":  {},
-	"blkio":   {},
-	"io":      {},
-	"devices": {},
+	"cpu":        {},
+	"cpuacct":    {},
+	"memory":     {},
+	"pids":       {},
+	"cpuset":     {},
+	"blkio":      {},
+	"io":         {},
+	"devices":    {},
+	"perf_event": {},
```

Comment:
Does this exist and work similarly in cgroups v2?

Reply:
It does exists and I believe it is similar but I have not investigated it yet.